consensus/src/experimental/buffer_manager.rs

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) The Diem Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use consensus_types::executed_block::ExecutedBlock;
+use diem_types::ledger_info::LedgerInfoWithSignatures;
+use futures::channel::{mpsc::UnboundedReceiver, oneshot};
+use std::{collections::LinkedList, sync::Arc};
+
+pub type ResetAck = ();
+pub fn reset_ack_new() -> ResetAck {}
+
+pub struct ResetEventType {
```

Comment:
this probably needs change later to carry a LI

Reply:
Right, we will change it later.